drools-parser/src/main/java/org/drools/parser/DRLVisitorImpl.java

```diff
@@ -140,13 +159,18 @@ public AccumulateImportDescr visitImportAccumulateDef(DRLParser.ImportAccumulate
 
     @Override
     public FunctionDescr visitFunctiondef(DRLParser.FunctiondefContext ctx) {
+        // function String functionA(String s, Integer i) {
+        //    foo();
+        // }
```

Comment:
Example syntax as a comment. Is this too much? But maybe helpful for new contributors.

Reply:
hi @tkobayas 
TBH I'm not so sure. On one side, I'm afraid that in the long term there could be more comments then actual code and, paradoxically, this could decrease readability. On the other side, it is slightly confusing because it is not clear (at least to me)  if the function must have exactly that signature, or if it can be any possible function.
Maybe for such low level details it would be better to create a sort of readme/manual/developer guide: wdyt ?

Reply:
Syntax details are written in documentation. The comments are just examples to help developers, but I agreed that it could decrease readability. Removed the syntax example comments. Thanks, @gitgabrio